### DISCUSSION TITLE
Alerts not Added in Calendar?

### DISCUSSION THREAD

**Discussion by knela96** [spatie/icalendar-generator#107] [Bugs] [answered]
Hi I've been trying to create some alerts by adding it on the event, to be notified a day previous the starting date. Seems that the .ics file is well formatted but after checking the google calendar event it does not add any notification on the event and don't get notified even having notifications turned on, on pc and mobile.

(I changed the MailTo in the following code just for privacy.)

```
$calendar = Calendar::create()
           ->productIdentifier('Kypma')
           ->event(function (Event $event) {
               $event->name('Kypma - '.$this->eventInfo->title)
                ->description($this->eventInfo->description)
                ->uniqueIdentifier('kypma'.$this->eventInfo->id) //Add Unique eventID
                ->status(EventStatus::confirmed())
                ->attendee(Auth::user()->mail, Auth::user()->name, ParticipationStatus::accepted()) //User Mail
                ->withoutTimezone()
                ->startsAt($this->eventInfo->start) //Event Start
                ->endsAt($this->eventInfo->end)  //Event Start
                ->alertMinutesBefore(5, 'test alert');
                ;
           })
           ;
       $calendar->appendProperty(TextProperty::create('METHOD', 'REQUEST'));

       return (new MailMessage)
           ->subject('Kypma - Evento "'.$this->eventInfo->title.'" añadido a tu calendario')
           //->markdown('emails.calendar.delete') //Modify view in views\emails\calendar\create.blade.php
           ->attachData($calendar->get(), 'event.ics', [
               'mime' => 'text/calendar; charset=UTF-8; method=REQUEST',
           ]);
```

BEGIN:VCALENDAR
VERSION:2.0
PRODID:Kypma
METHOD:REQUEST
BEGIN:VEVENT
UID:kypma47
DTSTAMP:20230125T211117Z
SUMMARY:Kypma - sd
DESCRIPTION:
STATUS:CONFIRMED
ATTENDEE;CN=Eric;PARTSTAT=ACCEPTED:MAILTO:mail@gmail.com
DTSTART:20230126T211117Z
DTEND:20230127T211117Z
BEGIN:VALARM
ACTION:DISPLAY
DESCRIPTION:test alert
TRIGGER:-PT5M
END:VALARM
END:VEVENT
END:VCALENDAR

**Comment by rubenvanassche** [comment, marked as answer]
I think Google calendar is not supporting this feature due to SPAM.

**Reply by knela96** [reply]
Do you know another way to make it work, appart from creating a scheduled mail to remind the user?

**Reply by rubenvanassche** [reply]
Hmm not that I'm aware of 😞
